<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dbg</class>
 <widget class="QWidget" name="dbg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>104</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dbg</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Log level</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_level"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_clr">
       <property name="text">
        <string>Clear log</string>
       </property>
       <property name="icon">
        <iconset resource="../qccf.qrc">
         <normaloff>:/images/reset.png</normaloff>:/images/reset.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_save">
       <property name="text">
        <string>Save log</string>
       </property>
       <property name="icon">
        <iconset resource="../qccf.qrc">
         <normaloff>:/images/log.png</normaloff>:/images/log.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="log">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>64</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qccf.qrc"/>
 </resources>
 <connections/>
</ui>
